.vscode/cspell.json

```diff
@@ -618,6 +622,50 @@
       "words": [
         "pysdkci"
       ]
+    },
+    {
+      "filename": "sdk/storage/azure-storage-file-share/**",
+      "words": [
+        "XSMB",
+        "rcwd",
+        "rcwdl",
+        "prevsharesnapshot",
+        "mydirectory",
+        "myfile",
+        "mydir",
+        "amet",
+        "consectetur",
+        "elit",
+        "adipiscing",
+        "pyacrstoragestorname",
+        "pyrmtstoragestorname",
+        "vhds",
+        "abcdefghijklmnop",
+        "filetoupdate",
+        "mydomain",
+        "vhds",
+        "utdir",
+        "myaccount",
+        "testid",
+        "filea",
+        "fileb"
+      ]
+    },
+    {
+      "filename": "sdk/storage/**/_shared/*.py",
```

Comment:
Maybe just make this `sdk/storage/**/_shared/**` and combine with the one below?

Reply:
I fixed both.